skiko/src/commonMain/kotlin/org/jetbrains/skia/CodePoint.kt

```diff
@@ -0,0 +1,51 @@
+package org.jetbrains.skia
+
+// Copied from CharHelpers.skiko.kt
```

Comment:
Since it's from another repo, it's not super relevant. I'd suggest remove or replace with full link

Reply:
Removed.